packages/@guidesmiths/cuckoojs-schematics/package.json

```diff
@@ -42,7 +42,7 @@
   "schematics": "./dist/collection.json",
   "scripts": {
     "build": "tsc -p tsconfig.json",
-    "postbuild": "../../../node_modules/.bin/copyfiles -a -u 1 ./src/collection.json ./src/*/schema.json ./src/*/files/** ./src/*/files/*/** dist/",
+    "postbuild": "../../../node_modules/.bin/copyfiles -a -u 1 ./src/collection.json ./src/*/schema.json ./src/*/files/** ./src/*/files/*/** ./src/*/files/*/*/** dist/",
```

Comment:
Do you know a way to be able to copy all the files under ./src/*/files  without having to add 3 different routes?

./src/*/files/**
./src/*/files/*/**
./src/*/files/*/*/**

Reply:
I tried several ways and none worked. Finally I was pragmatic and did that way, but I'm sure there is a way to do that. I'll have a look to it again

Reply:
done